crates/nu-command/src/system/exec.rs

```diff
@@ -11,7 +13,11 @@ impl Command for Exec {
     fn signature(&self) -> Signature {
         Signature::build("exec")
             .input_output_types(vec![(Type::Nothing, Type::Any)])
-            .required("command", SyntaxShape::String, "The command to execute.")
+            .rest(
+                "command",
+                SyntaxShape::OneOf(vec![SyntaxShape::Any, SyntaxShape::GlobPattern]),
```

Comment:
I think it should be changed to `SyntaxShape::OneOf(vec![SyntaxShape::GlobPattern, SyntaxShape::Any])` too.

Reply:
oops, thanks!